modules/favicon/common/index.js

```diff
@@ -1,8 +1,14 @@
+const exportedAssets = {};
+
 if (!__TEST__) {
   // Favicon.ico should not be hashed, since some browsers expect it to be exactly on /favicon.ico URL
   require('!file-loader?name=[name].[ext]!./assets/favicon.ico'); // eslint-disable-line
 
   // Require all files from assets dir recursively addding them into assets.json
   let req = require.context('!file-loader?name=[hash].[ext]!./assets', true, /.*/);
-  req.keys().map(req);
+  req.keys().map(key => {
+    exportedAssets[`${key.replace('./', '')}`] = req(key);
```

Comment:
I'm not sure this is enough though, maybe also use `humps.camelize` ?

Reply:
@larixer I don’t understand what you’re asking. The purpose of this line is to allow you to reference the assets by name later on. It is meant to be equivalent to how you can access assets on the server (notably in the file `website.tsx`.

The files are all prefixed with `./` by default ai this just removes that. Otherwise you just reference each file by whatever name it is on disk.